resources/schema/Operator.json

```diff
@@ -4,841 +4,521 @@
       "required": ["operator", "prefix", "value"],
       "type": "object"
     },
     {
-      "properties": {
-        "operator": {
-          "const": "prefix_not_equal_to"
-        }
-      },
+      "properties": { "operator": { "const": "prefix_not_equal_to" } },
       "required": ["operator", "prefix", "value"],
       "type": "object"
     },
     {
       "properties": {
-        "operator": {
-          "const": "present_on_multiple_rows_within"
-        }
+        "operator": { "const": "present_on_multiple_rows_within" }
       },
       "required": ["operator", "within"],
       "type": "object"
     },
     {
-      "properties": {
-        "operator": {
-          "const": "references_correct_codelist"
-        }
-      },
+      "properties": { "operator": { "const": "references_correct_codelist" } },
       "required": ["operator", "value"],
       "type": "object"
     },
     {
       "properties": {
-        "operator": {
-          "const": "shares_at_least_one_element_with"
-        }
+        "operator": { "const": "shares_at_least_one_element_with" }
       },
       "required": ["operator", "value"],
       "type": "object"
     },
     {
       "properties": {
-        "operator": {
-          "const": "shares_exactly_one_element_with"
-        }
+        "operator": { "const": "shares_exactly_one_element_with" }
       },
       "required": ["operator", "value"],
       "type": "object"
     },
     {
-      "properties": {
-        "operator": {
-          "const": "shares_no_elements_with"
-        }
-      },
+      "properties": { "operator": { "const": "shares_no_elements_with" } },
       "required": ["operator", "value"],
       "type": "object"
     },
     {
-      "properties": {
-        "operator": {
-          "const": "shorter_than"
-        }
-      },
+      "properties": { "operator": { "const": "shorter_than" } },
       "required": ["operator", "value"],
       "type": "object"
     },
     {
-      "properties": {
-        "operator": {
-          "const": "shorter_than_or_equal_to"
-        }
-      },
+      "properties": { "operator": { "const": "shorter_than_or_equal_to" } },
       "required": ["operator", "value"],
       "type": "object"
     },
     {
-      "properties": {
-        "operator": {
-          "const": "starts_with"
-        }
-      },
+      "properties": { "operator": { "const": "starts_with" } },
       "required": ["operator", "value"],
       "type": "object"
     },
     {
       "properties": {
-        "operator": {
-          "const": "suffix_equal_to"
-        }
+        "operator": { "const": "split_parts_have_equal_length" }
       },
-      "required": ["operator", "suffix", "value"],
+      "required": ["operator"],
       "type": "object"
     },
     {
       "properties": {
-        "operator": {
-          "const": "suffix_is_contained_by"
-        }
+        "operator": { "const": "split_parts_have_unequal_length" }
```

Comment:
It looks like we are maintaining alphabetic order. These should go before "starts_with"